command/aarelayer/service/aa_server.go

```diff
@@ -0,0 +1,127 @@
+package service
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// AARelayerRestServer represents the service for handling account abstraction transactions
+type AARelayerRestServer struct {
+	pool         AAPool
+	state        AATxState
+	verification AAVerification
+	server       *http.Server
+}
+
+func NewAARelayerRestServer(pool AAPool, state AATxState, verification AAVerification) *AARelayerRestServer {
+	return &AARelayerRestServer{
+		pool:         pool,
+		state:        state,
+		verification: verification,
+	}
+}
+
+// SendTransaction handles the /v1/sendTransaction endpoint
+func (s *AARelayerRestServer) sendTransaction(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+
+		return
+	}
+
+	var tx AATransaction
+
+	if err := json.NewDecoder(r.Body).Decode(&tx); err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+
+		return
+	}
+
+	if err := s.verification.Validate(&tx); err != nil {
+		writeMessage(w, http.StatusBadRequest, err.Error())
+
+		return
+	}
+
+	// put to DB
+	stateTx, err := s.state.Add(&tx)
+	if err != nil {
+		writeMessage(w, http.StatusInternalServerError, err.Error())
+
+		return
+	}
+
+	// put to pool
+	s.pool.Put(stateTx.ID, &tx)
+
+	writeOutput(w, map[string]string{"uuid": stateTx.ID})
+}
+
+// GetTransactionReceipt handles the /v1/getTransactionReceipt/{uuid} endpoint
+func (s *AARelayerRestServer) getTransactionReceipt(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodGet {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+
+		return
+	}
+
+	uuid := r.URL.Path[len("/v1/getTransactionReceipt/"):]
```

Comment:
Would be great to move this to a constant

Reply:
Fixed fixed https://github.com/0xPolygon/polygon-edge/pull/1252/commits/f8527534f60da34f423360164a5bfd3a2cbf29d9